api/schema_system.go

```diff
@@ -117,3 +162,24 @@ func UnmarshalNamingRulePayloadFormat(payload string) (*regexp.Regexp, error) {
 	}
 	return format, nil
 }
+
+// UnmarshalTemplateRulePayload will unmarshal payload to TemplateRulePayload and compile it as list of TemplateRuleSection.
+// For example, "hard_code_{{table}}_{{column}}_end" will return
+// "hard_code_{{table}}_{{column}}_end", ["{{table}}", "{{column}}"]
```

Comment:
There is no `TemplateRuleSection`. We need to modify this comment.

Reply:
Updated.